packages/client/src/layers/network/createNetworkLayer.ts

```diff
@@ -49,9 +52,16 @@ export async function createNetworkLayer(config: GameConfig) {
   }
 
   function uploadSound(soundUri: string) {
-    systems["system.UploadSound"].executeTyped(soundUri);
+    systems["system.UploadSound"].executeTyped(BigNumber.from(network.connectedAddress.get()), soundUri);
```

Comment:
Entity ID was missing before. That's why it wouldn't work @0xPetra

Reply:
This is also how we "create" entities. We just have a system execute with an entity ID. In the demo, the entity ID is hardcoded to a connection ID. But we want to create a convention for creating Sound, Track, and Sample entity IDs